app/src/main/java/protect/card_locker/ManageGroupLoyaltyCard.java

```diff
@@ -0,0 +1,168 @@
+package protect.card_locker;
+// Was thinking about extending LoyaltyCard but this makes things more modular..?
```

Comment:
I don't really agree this makes stuff more modular. The only thing you added here seems to be checking if it's in a group, I would rather the ManageGroupView just does the check itself like LoyaltyCardView does:

https://github.com/TheLastProject/Catima/blob/4f5cd8791282b585bfcc885ed268a085dd117b10/app/src/main/java/protect/card_locker/LoyaltyCardViewActivity.java#L447